packages/core/src/components/HierarchyPicker/Hierarchy/helpers.ts

```diff
@@ -0,0 +1,24 @@
+import { ItemShape } from '../types';
+
+export function allChildrenReadonly(parent: ItemShape): Boolean {
```

Comment:
`boolean` not `Boolean`. Be sure to apply this everywhere as I've seen it a few times.

Reply:
not added by me but 👍